chore: add issue and pr templates (#13)
eludris/pengin: .github/PULL_REQUEST_TEMPLATE.md

## Description

<!-- Explain what this Pull Request changes -->

<!-- Link any issues if applicable.
Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
-->

<!-- Uncomment based on the type of your changes below -->

<!--
## This is a **Logic Change**
- [ ] Docs have been updated to reflect these changes if necessary.
- [ ] Changes have been tested.
-->

<!--
## This is a **UI Change**

> - [ ] This has been previewed and's output is as intended

Reviewer: ```suggestion
- [ ] This has been previewed and looks as intended
```
